source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -0,0 +1,264 @@
+.. _dual-boot-linux:
+
+Dual Boot |CL-ATTR| with Another Grub-based Linux\* Distro
+##########################################################
+
+In this tutorial, we show how to install another grub-based Linux\* distro 
+alongside |CL|.  To do so, we resize the existing Linux root partition to 
+make enough room to install |CL|. Then we configure the |CL| bootloader, 
+:command:`systemd-boot`, which enables you to dual-boot |CL| and an existing
+Linux distro.  Although we use Ubuntu\* 19.04 Desktop as the example here, 
+these instructions also work for other distros such as Mint Linux, Kubuntu\*, Fedora\*, CentOS\*, among others.
+
+Prerequisites
+*************
+
+* Ubuntu 19.04 Desktop is already installed. 
+* There is no unallocated space available, but there is free space 
+  on the Ubuntu root partition.
+* Follow the Preliminary steps that follow. 
+
+.. include:: ../../get-started/bare-metal-install-desktop.rst
+   :start-after: preliminary-steps-install-desktop:
+   :end-before: install-on-target-start:
+
+Use Select Installation Media
```

Comment:
This heading "Use Select Installation Media" doesn't make sense to me. The primary objective of this section is to install Clear Linux, so the title should be "Install Clear Linux Desktop".